pageserver/src/repository.rs

```diff
@@ -69,6 +71,12 @@ pub trait Timeline {
     /// Truncate relation
     fn put_truncation(&self, rel: RelTag, lsn: Lsn, nblocks: u32) -> Result<()>;
 
+    /// Put raw data
+    fn put_raw_data(&self, key: RepositoryKey, data: &[u8]) -> Result<()>;
```

Comment:
How this different from put_page_image()?

Reply:
wit_lsn is needed to implement get_range() method which was moved our of rocksdb code.